### PR TITLE
Install flipmove animation dependency, animate item list.

### LINKED ISSUE #45: As a user, I would like the sorting of an item upon purchase or deletion to be a smooth user experience.
Currently, when a user purchases or deletes an item, the list resorts immediately, causing a jarring visual effect. It can be hard to follow what has happened.

### PR DESCRIPTION
**Pull request by chriskorsak**
## Description

In this PR, we have installed a dependency called FlipMove, which allows animating a list upon reorder. When a user purchases or deletes an item or filters the item list, the list animates, allowing the user to more easily follow what is happening. After running into difficulties with preventing the item list from re-sorting after purchase/delete, we feel that providing visual cues using this animation instead is a good compromise.

## Related Issue

closes #45 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

### Short video clip of feature:
Screen_Recording_2022-02-22_at_6_25_53_PM_AdobeCreativeCloudExpress

## Testing Steps / QA Criteria

- Pull down branch
- Run npm install to install flipmove dependency
- Run npm start to start app
- Filter the list, purchase, or delete an item to see animation

### CODE REVIEW

**Review by cynthiaeddy — APPROVED** [review]
nice solution

**Review by k-monnik — APPROVED** [review]
This is a cool effect, clever way to address the purchase/sorting behavior.

**Review by skylerwshaw — APPROVED** [review]
Slick solution! I like how this comes from a [Ryan Florence](https://twitter.com/ryanflorence) (React founder) [hack project](http://joshwcomeau.github.io/react-flip-move/#features)!
